pkg/controllers/csi/provisioner/controller.go

```diff
@@ -229,25 +233,25 @@ func (provisioner *OneAgentProvisioner) collectGarbage(ctx context.Context, requ
 	return result, nil
 }
 
-func (provisioner *OneAgentProvisioner) provisionCodeModules(ctx context.Context, dk *dynatracev1beta2.DynaKube, tenantConfig *metadata.TenantConfig) error {
+func (provisioner *OneAgentProvisioner) provisionCodeModules(ctx context.Context, dk *dynatracev1beta2.DynaKube, tenantConfig *metadata.TenantConfig) (bool, error) {
```

Comment:
I think since you changed the method's signature, and now it returns a bool, we should consider renaming it to something like `isProvisionCodeModules`. Then you magic constants like true / false will be more clear.

Reply:
well the bool indicates if we need a requeue or not in case of an error, which is something different imho than if the codeModules got provisioned successfully

Reply:
@andriisoldatenko Fixed with naming the return types :)